pkg/services/sqlstore/dashboard_provisioning.go

```diff
@@ -17,6 +18,23 @@ type DashboardExtras struct {
 	Value       string
 }
 
+func GetProvisionedDataByDashboardId(cmd *models.GetProvisionedDashboardByDashboardId) error {
+	result := &models.DashboardProvisioning{}
+
+	exist, err := x.Where("dashboard_id = ?", cmd.DashboardId).Get(result)
```

Comment:
Is orgId required as well or maybe I'm missing something?

Reply:
That table does not have OrgID at the moment. Since all dashboards have their own Id, regardless of org it's not really needed.

Reply:
Cool :+1: